onadata/apps/api/tests/viewsets/test_xform_viewset.py

```diff
@@ -4711,6 +4771,11 @@ def test_csv_exports_w_images_link(self):
                 "uuid1",
                 media_file,
             )
+            try:
+                cmd = f"rm {settings.MEDIA_ROOT}*/attachments/*/{media_file}"
```

Comment:
Why do we need this?

Reply:
This was an attempt at fixing a flacky test, removed!